configure.py

```diff
@@ -419,6 +427,18 @@ def MatchingFor(*versions) -> bool:
 
 config.warn_missing_config = True
 config.warn_missing_source = False
+config.precompiled_headers = [
+    {
+        "source": "d/dolzel_framework.pch",
```

Comment:
I would name the framework one like this so the vibration symbols match in the debug build.
```suggestion
        "source": "d/dolzel.pch",
```